ui/apps/platform/src/routePaths.ts

```diff
@@ -408,6 +414,7 @@ const vulnerabilitiesPathToLabelMap = {
     [vulnerabilitiesBasePath]: 'Vulnerabilities',
     [vulnerabilitiesWorkloadCvesPath]: 'Workload CVEs',
     [vulnerabilityReportsPath]: 'Vulnerability Reporting',
+    [exceptionManagementPath]: 'Exception Management',
```

Comment:
Ditto comment by Dave to render `PageTitle` element, which makes this unneeded, although consistent.